.travis.yml

```diff
@@ -48,14 +43,8 @@ matrix:
           compiler: gcc-5
           env: CONFIG_OPTS="no-asm --strict-warnings -fno-sanitize-recover -fsanitize=address -fsanitize=undefined enable-rc5 enable-md2"
         - os: linux
-          compiler: clang-3.6
-          env: CONFIG_OPTS="no-engine"
-        - os: linux
-          compiler: gcc
-          env: CONFIG_OPTS="no-engine"
-        - os: linux
-          compiler: gcc-5
-          env: CONFIG_OPTS="no-engine"
+          compiler: clang
+          env: CONFIG_OPTS="no-engine" BUILDONLY="yes"
```

Comment:
Well, since you ask - it depends :) Compilers get smarter, which isn't always better if you want to test build issues. I even have a recent example:
https://github.com/openssl/openssl/commit/5b7af0dd6c9315ca76fba16813b66f5792c7fe6e

 In this particular case, I don't see how it could matter either way.